models/noise_generator.cpp

```diff
@@ -264,7 +338,9 @@ nest::noise_generator::update( Time const& origin,
         *it = P_.mean_
           + std::sqrt( P_.std_ * P_.std_ + S_.y_1_ * P_.std_mod_ * P_.std_mod_ )
             * V_.normal_dev_( kernel().rng_manager.get_rng( get_thread() ) );
+        S_.I_ += *it;
       }
+      S_.I_ /= B_.amps_.size();
 
```

Comment:
Same problem as for AC above. When ON, the current = P_.amp_. But for the OFF period(s), in the absence of we would need to perform a time check to set current = 0 within get_I_(). Figure in above comment for AC.